### PR TITLE
[TM-13] Add user registration

### PR DESCRIPTION
**Pull request by faliszewskii**

### CODE REVIEW

File: src/front/ticketer/lib/pages/page_user_register.dart [hunk 1]
```diff
@@ -1,51 +1,237 @@
...
-    });
+      showDialog(
+        context: context,
+        builder: (context) {
+          return AlertDialog(
+            title: const Text("Thank you"),
+            content: const Text("You can sign in into the account now"),
+            actions: [
+              ElevatedButton(
+                onPressed: () => {
```
Comment (gorkaj) [inline src/front/ticketer/lib/pages/page_user_register.dart:139]: Add `Navigator.pop(context)` before pushReplacement to pop modal first

File: src/front/ticketer/lib/pages/page_user_register.dart [hunk 2]
```diff
@@ -1,51 +1,237 @@
...
+                  Navigator.pushReplacement(
```
Comment (bjadczak) [inline src/front/ticketer/lib/pages/page_user_register.dart:140]: This shouldn't be pushReplacement but rather pop

**Review by bjadczak — CHANGES_REQUESTED** [review]

File: src/front/ticketer/lib/pages/page_user_register.dart (hunk 1 above)
Comment (faliszewskii) [reply src/front/ticketer/lib/pages/page_user_register.dart]: resolved

File: src/front/ticketer/lib/pages/page_user_register.dart (hunk 2 above)
Comment (faliszewskii) [reply src/front/ticketer/lib/pages/page_user_register.dart]: resolved

**Pull request closed without merging**